Framework/PythonInterface/test/testhelpers/__init__.py

```diff
@@ -38,7 +40,11 @@ def create_algorithm(name, **kwargs):
         kwargs - A dictionary of property name:value pairs
     @returns The algorithm handle
     """
-    alg = mantid.api.AlgorithmManager.createUnmanaged(name)
+    if 'Version' in kwargs:
```

Comment:
True, but from the other hand you are repeating here the default value, which might change theoretically, and if this ever happens, one should take care to change here as well. In principle, this code should not know what is the default value of version in the algorithm manager, so I am not entirely sure it is worth it.